app/services/calculators/property_calculator.rb

```diff
@@ -46,7 +51,7 @@ def calculate_smod_disregard(property, net_equity, smod_level)
       end
 
       def calculate_assessed_equity(net_equity, main_home_equity_disregard)
-        [net_equity - main_home_equity_disregard, 0].max
+        [net_equity - main_home_equity_disregard, 0.0].max
```

Comment:
I'm worried that I might have broken a situation where client has negative equity in a house - because this is doing the calculation quite early in the process (a bit too early maybe?) - we sum these values to produce the 'total capital'

Reply:
@starswan are you still worried about this? Shall we add a test to prove it doesn't change behaviour in this situation?

Reply:
@patrick-laa I've spoken to Mike H, and apparently this is actually the correct behaviour, so it turns out that I have fixed a bug rather than created one